readthedocs/doc_builder/base.py

```diff
@@ -63,7 +65,12 @@ def move(self, **__):
             if os.path.exists(self.target):
                 shutil.rmtree(self.target)
             log.info('Copying %s on the local filesystem', self.type)
-            shutil.copytree(self.old_artifact_path, self.target)
+            log.info('Ignoring patterns %s', self.ignore_patterns)
+            shutil.copytree(
+                self.old_artifact_path,
+                self.target,
+                ignore=shutil.ignore_patterns(*self.ignore_patterns)
```

Comment:
This is from https://docs.python.org/2/library/shutil.html#shutil.ignore_patterns